### PR TITLE
Add profile viewer method

### PR DESCRIPTION
**Pull request by lalmei**
## Description

Moved viewer to within `src` and added a methods to open browser tab with the static html

The following method can use to load the viewer:
```
from  whylogs.viz import profile_viewer

profile_viewer()
```
opens a browser tab with the profile viewer ready to be loaded with profile json.

### General Checklist

* [x] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [x] Conform by the style guides, by using formatter
* [x] Documentation updated
* [x] (optional) Please add a label to your PR

### CODE REVIEW

**Comment by andyndang** [comment]
Looks good. I do have concern for bundling a bunch of font files with whylogs, but we can revisit the decision later

**Review by andyndang — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -90,6 +90,38 @@ with session.logger(dataset_name="my_dataset") as logger:
...
+
+This will open a viewer on your default browser where you can load a profile json summary, using the `Select JSON profile` button:
+
+![viewer_preload](images/html_viewer.png)
+
+Once the json is selected you can view your profile's features and associated and statistics.
+
+![viewer_loaded](images/html_viewer_loaded.png)
+
 whyLogs collects approximate statistics and sketches of data on a column-basis into a statistical profile. These metrics include:
```
Comment (andyndang) [inline README.md]:
```suggestion
whylogs collects approximate statistics and sketches of data on a column-basis into a statistical profile. These metrics include:
```

File: README.md [hunk 2]
```diff
@@ -90,6 +90,38 @@ with session.logger(dataset_name="my_dataset") as logger:
     #images
     logger.log_images("path/to/image.png")
 ```
+
```
Comment (andyndang) [inline README.md:87]: Add a section for visualizing the profile here

**Review by andyndang — CHANGES_REQUESTED** [review]
* It's really hard to see what's going on in the screenshot (too big)
* The screenshots being added before the `whylogs collects approximate statistics and sketches...` section make the structure strange/breaks the flow

File: README.md (hunk 1 above)
Comment (lalmei) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (lalmei) [reply README.md:87]: done

Comment (andyndang) [reply README.md:87]: I think we should move the paragraph starting with "whylogs collects approximate statistics and sketches..." right after the first code example

File: README.md [hunk 3]
```diff
@@ -90,17 +84,50 @@ with session.logger(dataset_name="my_dataset") as logger:
     #images
     logger.log_images("path/to/image.png")
 ```
-whyLogs collects approximate statistics and sketches of data on a column-basis into a statistical profile. These metrics include:
+
+### Multiple profile plots
```
Comment (andyndang) [inline README.md]:
```suggestion
### Multiple Profile Plots
```

ensure consistent capitalization style

**Review by andyndang — CHANGES_REQUESTED** [review]

**Comment by andyndang** [comment]
I think we should move the paragraph starting with "whylogs collects approximate statistics and sketches..." right after the first code example

**Comment by lalmei** [comment]
+1 @andyndang  moved it

File: README.md [hunk 4]
```diff
@@ -90,22 +84,56 @@ with session.logger(dataset_name="my_dataset") as logger:
     #images
     logger.log_images("path/to/image.png")
 ```
-whyLogs collects approximate statistics and sketches of data on a column-basis into a statistical profile. These metrics include:
+
+whylogs collects approximate statistics and sketches of data on a column-basis into a statistical profile. These metrics include:
 
 - Simple counters: boolean, null values, data types.
 - Summary statistics: sum, min, max, variance.
```
Comment (lalmei) [inline README.md]:
```suggestion
- Summary statistics: sum, min, max, median, variance.
```

File: README.md [hunk 5]
```diff
@@ -90,22 +84,56 @@ with session.logger(dataset_name="my_dataset") as logger:
...
 - Unique value counter or cardinality: tracks an approximate unique value of your feature using HyperLogLog algorithm.
 - Histograms for numerical features. whyLogs binary output can be queried to with dynamic binning based on the shape of your data.
```
Comment (lalmei) [inline README.md:93]:
```suggestion
- Histograms for numerical features. whyLogs binary output can be queried to with dynamic binning based on the shape of your data.

```

File: README.md [hunk 6]
```diff
@@ -90,22 +84,56 @@ with session.logger(dataset_name="my_dataset") as logger:
...
 
-<img align="center" src="images/Whylabs-Dots-Light-Bg.png" width="250"><img align="center" src="images/Whylabs-Dots-Light-Bg.png" width="250"><img align="center" src="images/Whylabs-Dots-Light-Bg.png" width="250">
+### Multiple Profile Plots
+
+To view your logger profiles you can use, methods within `whylogs.viz`: 
+
+```python
+vizualization = ProfileVisualizer()
+vizualization.set_profiles([profile_day_1, profile_day_2])
+figure= my_ viz.plot_distribution("<feature_name>")
```
Comment (lalmei) [inline README.md]:
```suggestion
figure= vizualization.plot_distribution("<feature_name>")
```

File: README.md [hunk 7]
```diff
@@ -90,22 +84,56 @@ with session.logger(dataset_name="my_dataset") as logger:
...
+figure.savefig("/my/image/path.png")
+```
+
+Individual profiles are saved to disk, AWS S3, or WhyLabs API, automatically when loggers are closed, per the configuration found in the Session configuration.
+
+Current active loggers can be loaded from memory with:
```
Comment (lalmei) [inline README.md]:
```suggestion
Current profiles from active loggers can be loaded from memory with:
```

**Review by andyndang — APPROVED** [review]
LGTM. Though we probably want to store the gif file in the repo instead of S3

**Pull request merged**